xbmc/video/Teletext.cpp

```diff
@@ -32,11 +32,6 @@
 #include "filesystem/SpecialProtocol.h"
 #include "guilib/GraphicContext.h"
 
-#if HAVE_SDL_VERSION == 1
-#include <SDL/SDL_stdinc.h>
-#elif HAVE_SDL_VERSION == 2
-#include <SDL2/SDL_stdinc.h>
-#else
```

Comment:
great. 
maybe we can drop SDL_memcpy/memset completely on a next step. I would guess that replacing it is not too hard.